helm/volsync/README.md

```diff
@@ -86,6 +86,8 @@ Please see https://volsync.readthedocs.org for documentation.
 The following parameters in the chart can be configured, either by using `--set`
 on the command line or via a custom `values.yaml` file.
 
+- `manageCRDs`: true
+  - Whether the chart should install/upgrade/remove the VolSync CRDs
```

Comment:
should this comment not mention "remove" for the VolSync CRDs?

Reply:
Good catch. fixed